common/static/coffee/src/discussion/discussion_module_view.coffee

```diff
@@ -10,10 +10,11 @@ if Backbone?
       "click .discussion-paginator a": "navigateToPage"
 
     page_re: /\?discussion_page=(\d+)/
-    initialize: ->
+    initialize: (options) ->
       @toggleDiscussionBtn = @$(".discussion-show")
       # Set the page if it was set in the URL. This is used to allow deep linking to pages
       match = @page_re.exec(window.location.href)
+      @context = options.context or "course"  # allowed values are "course" or "standalone"
```

Comment:
Sure thing, let's discuss it. I can imagine that we'll have lots of team-specific customizations, so having a single setting that can drive them all seems more flexible. I'm not sure I like the idea of introducing very targeted properties for individual capabilities that we want, but I'll entertain the idea.